css/styles.css

```diff
@@ -0,0 +1,118 @@
+body {
+  background-color: #161616;
+}
+.logo-name {
+  color: #990600;
+  font-size: 3rem;
+  font-family: monospace;
+  text-align: center;
+  margin: 0;
+}
+.logo-image {
+  width: 40%;
+  display: block;
+  margin: 0 auto;
+}
+nav a {
+  display: block;
+  font-size: 2rem;
+  text-decoration: none;
+  text-align: center;
+  padding: 1rem 0;
+  background-color: #990600;
+  color: #fff;
+  font-weight: bold;
+  transition: 0.5s ease;
+}
+nav a:hover {
+  background-color: #fff;
+  color: #990600;
+}
+@media (min-width: 800px) {
+  header {
+    width: 95%;
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+  nav a {
+    padding: 1.5rem;
+    border-radius: 0.8rem;
+  }
+}
+.pc-sell {
+  width: 95%;
+  margin: 0 auto;
+}
+.pc-image {
+  display: block;
+  width: 50%;
+  margin: 1rem auto 0 auto;
+  padding: 1rem 0;
+}
+.pc-info {
+  font-size: 1.6rem;
+  color: #fff;
+  text-align: center;
+  background-color: #990600;
+  padding: 1rem;
+  border-radius: 1rem;
+}
+.pc-price {
```

Comment:
for the price, it would look better if it would has some styles like a bolder font,